src/moi_wrapper.jl

```diff
@@ -21,19 +21,35 @@ julia> x = model.add_variable(model)
 julia> model.add_constraint(model, ...)
 ```
 """
-function diff_optimizer(optimizer_constructor)::Optimizer
-    optimizer =
-        MOI.instantiate(optimizer_constructor; with_bridge_type = Float64)
+function diff_optimizer(
+    optimizer_constructor;
+    method = nothing,
+    with_parametric_opt_interface::Bool = false,
+    with_bridge_type = Float64,
+    with_cache::Bool = true,
+)
+    optimizer = MOI.instantiate(
+        optimizer_constructor;
+        with_bridge_type = with_bridge_type,
```

Comment:
```suggestion
        with_bridge_type,
```

Reply:
This needs a lower bound in Julia, right? 1.6 is enough?